integration-tests/contract-estimation-tests.spec.ts

```diff
@@ -1,13 +1,13 @@
-import { DEFAULT_FEE, MANAGER_LAMBDA, TezosToolkit, Protocols } from "@taquito/taquito";
+import { DEFAULT_FEE, MANAGER_LAMBDA, Protocols, TezosToolkit } from "@taquito/taquito";
 import { Contract } from "@taquito/taquito";
 import { CONFIGS } from "./config";
 import { originate, originate2, transferImplicit2 } from "./data/lambda";
 import { ligoSample } from "./data/ligo-simple-contract";
 import { managerCode } from "./data/manager_code";
 
-CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) => {
+CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, rpc, protocol }) => {
   const Tezos = lib;
-  const limanet = (protocol === Protocols.PtLimaPtL) ? test : test.skip;
+  const Mumbainet = protocol === Protocols.PtMumbai2 ? it : it.skip;
```

Comment:
nitpick, `Mumbainet` is a function, so should be lowercase

Reply:
right went based off a different one that was capitalized. will change the other too

Reply:
actually already removed the other version of it from `Limanet` 😓